sdk/burdock/query/sql/private/query.py

```diff
@@ -149,7 +152,7 @@ def execute_ast(self, query):
         """
         if isinstance(query, str):
             raise ValueError("Please pass ASTs to execute_typed.  To execute strings, use execute.")
-        trs = self.execute_ast_exact(query)
```

Comment:
this seemed like a bigger bug than before. Did the stochastic tester catch this/

Reply:
Is introducing 0 as the base value valid for DP? It is implied to be low anyways with the null return I suppose

Reply:
That bug wasn't getting hit because there were no tests for it.  The other bug requiring setting NULL == 0 was indeed caught by the stochastic tester.  Setting null equal to 0 is valid (and required) in the case of counts, and I *think* in the case of other statistics, but I will need to add more tests